variable.tf

```diff
@@ -0,0 +1,49 @@
+variable "name" {
+  description = "name of the msk cluster"
+  default     = ""
```

Comment:
I don't think you can have this be the default since I don't think you can have the name of an MSK cluster be empty. I think this should be required